cartridges/app_custom/cartridge/templates/default/product/productTags.isml

```diff
@@ -0,0 +1,13 @@
+<isset name="counter" value="${1}">
+
+<isif condition = "${pdict.product.tags.length  > 0}">
+        <isloop items="${pdict.product.tags}" var="tags" status = "status">
+	        <isif condition="${tags.length <= 6}">
```

Comment:
you can use <iscontinue/> if length > 6, so no need to wrap the whole logic in condition